doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -270,6 +270,44 @@ Insufficiently restrictive variance annotations lead to errors:
 
    Fail Polymorphic Cumulative Record bad@{*a} := {p : Type@{a}}.
 
+.. example:: Demonstration of universe variances
+
+   Here is an example demonstating the behavior of various universe variances.
+
+   .. coqtop:: all
```

Comment:
```suggestion
   .. coqtop:: in
```

Let's omit the output for a bunch of the dull stuff at the beginning--I think the example is too long.